library/ix-dev/community/jenkins/templates/_validation.tpl

```diff
@@ -0,0 +1,44 @@
+{{- define "jenkins.validation" -}}
+  {{- if and (not .Values.jenkinsNetwork.http) (not .Values.jenkinsNetwork.https) -}}
+    {{- fail "Jenkins - At least one of [HTTP, HTTPS] must be enabled." -}}
```

Comment:
Can both be enabled at the same time?

Reply:
Just to be sure both http and https work when both options are checked?

Reply:
Yes, that's correct

Reply:
In practice it is better to not allow http access when https is enabled, is there specific motivation for this setup?

Reply:
I agree, only reason I've "allowed" it is if there are any other services that need to interact with Jenkins and do not like https.

Reply:
Are there any such services at the moment?

Reply:
I'm not aware of any.

Reply:
@stavros-k let's remove this behaviour in that case then ;)